test/export/StructureDefinitionExporter.test.ts

```diff
@@ -426,7 +426,7 @@ describe('StructureDefinitionExporter', () => {
     expect(baseSubject.type).toHaveLength(1);
     expect(baseSubject.type).toEqual([
       {
-        code: 'Reference',
+        _code: 'Reference',
```

Comment:
OK... so when we need to assert a `code` and `profile` or `targetProfile`, if we decide to take a different approach, we need to account for that.  So I guess based on the three options I outlined above, this might look like:
```ts
assertType(
  baseSubject.type[0],
  'Reference',
  null, // this arg is probably for 'profile'
  [
    'http://hl7.org/fhir/StructureDefinition/Patient',
    'http://hl7.org/fhir/StructureDefinition/Group',
    'http://hl7.org/fhir/StructureDefinition/Device',
    'http://hl7.org/fhir/StructureDefinition/Location'
  ]
);
```
or this:
```ts
assertType(
  baseSubject.type[0],
  {
    code: 'reference',
    targetProfile: [
      'http://hl7.org/fhir/StructureDefinition/Patient',
      'http://hl7.org/fhir/StructureDefinition/Group',
      'http://hl7.org/fhir/StructureDefinition/Device',
      'http://hl7.org/fhir/StructureDefinition/Location'
    ]
  }
);
```
or
```ts
expect(baseValue.type[0]).toEqual(
  new ElementDefinitionType('Quantity')
    .withTargetProfiles(
      'http://hl7.org/fhir/StructureDefinition/Patient',
      'http://hl7.org/fhir/StructureDefinition/Group',
      'http://hl7.org/fhir/StructureDefinition/Device',
      'http://hl7.org/fhir/StructureDefinition/Location'
    )
);
```

Reply:
Maybe we should bring this up for discussion w/ the team.  But I think I'm starting to feel like maybe that last option is best -- as it also most closely resembles what is _actually_ expected.